src/register-org/components/registered-with-regulator/registered-with-regulator.component.spec.ts

```diff
@@ -16,7 +16,11 @@ describe('RegisteredWithRegulatorComponent', () => {
     hasPBA: null,
     contactDetails: null,
     hasRegisteredWithRegulator: null,
-    services: []
+    services: [],
+    address: null,
+    organisationType: null,
+    regulatoryOrgType: null,
+    reulatorRegisteredWith: null
```

Comment:
@DavidJayakumar - looks like a typo here; perhaps you meant `regulatorRegisteredWith`?